sys/include/net/ng_netconf.h

```diff
@@ -119,8 +119,12 @@ typedef enum {
      * @note not all transceivers may support this interrupt
      */
     NETCONF_OPT_TX_END_IRQ,
-    NETCONF_OPT_AUTOCCA,            /**< en/disable to check automatically
-                                         before sending the channel is clear. */
+    NETCONF_OPT_CSMA,               /**< en/disable to check automatically
```

Comment:
> We should distinguish between the terms CCA and CSMA. CCA only senses the channel for an ongoing transmission. CSMA/CA, however, is the complete process that includes backoff handling. CCA is rather a part of the CSMA/CA process.

That's exactly what I've meant. Thanks for the elaboration, @jremmert-phytec-iot. I was a little too tired yesterday to express myself clearer.